.travis.yml

```diff
@@ -36,4 +44,6 @@ install:
   - export PATH=$PWD/travis-phantomjs:$PATH
 
 script:
-  - ember try $EMBER_TRY_SCENARIO test
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
@JoshSmith - it was recommended by the diff provided by `ember init` when I upgraded the add-on.

I did a bit of digging and found this:

* https://github.com/ember-cli/ember-cli/pull/5606
* https://github.com/ember-cli/ember-cli/commit/0111a51ed1aeb4bc8632220bad7b81011640a5cb

Looks to just be cutting out unnecessary work based on state.